generators/markup/files/page.js

```diff
@@ -0,0 +1,61 @@
+
+import { createElement, PropTypes, Component } from 'react';
+
+function script({ path, content, type, id }, i) {
+  return path ?
+    <script type={type} src={path} key={i}/> :
+    <script type={type} key={i} id={id} dangerouslySetInnerHTML={{
+      __html: content
+    }}/>;
+}
+
+function style({ path, type, content }, i) {
```

Comment:
I think these are very specific to where you're using them here. Feels like glue code more than library code to me